docs/getting-started.md

```diff
@@ -1,22 +1,27 @@
 # Getting Started
 
-Connect to Cube AI deployed instance on port `:6193`. Login with your username and password. If you can't log in ask the instance administrator to create a new user.
+Before logging in or interacting with Cube AI, you need to make sure that Cube AI has been deployed and is running. Deployment can be done by the instance administrator or by you if you have the necessary access permissions. Therefore, to Connect:
+
+1. Ensure that Cube AI is deployed and running on your instance.
+2. Connect to the deployed Cube AI instance on port `6193` using your login credentials (username and password).
```

Comment:
Can you explain why this port comes from? If this is Oolama specific, then mention this. If this is ours then explain that it is user-definable through ENV and that this is just a default.